extensions/objects/templates/FilepathEditor.js

```diff
@@ -161,6 +161,12 @@ oppia.directive('filepathEditor', [
           $scope.filepaths = response.data.filepaths;
           $scope.filepathsLoaded = true;
         });
+        $scope.width = null;
```

Comment:
Alright, so the user enters a number in the modal but checks the preview in the RTE? Hm, but we could use this for now, it allows the user to change the width atleast.